pkg/kubectl/resource_printer.go

```diff
@@ -1476,6 +1476,9 @@ func printNode(node *api.Node, w io.Writer, options PrintOptions) error {
 	if node.Spec.Unschedulable {
 		status = append(status, "SchedulingDisabled")
 	}
+	if role := findNodeRole(node); role != "" {
```

Comment:
This was one of the my open questions here.  Should I surface it as a column and add that to the default output instead?  I do think that would be cleaner if that's OK...